llama_index/indices/service_context.py

```diff
@@ -111,6 +113,21 @@ def from_defaults(
             )
             chunk_size = chunk_size_limit
 
+        if isinstance(embed_model, str):
+            if embed_model != "local":
+                raise ValueError(
+                    "embed_model must be str 'local' or of type BaseEmbedding"
+                )
+            try:
+                from langchain.embeddings import HuggingFaceEmbeddings
+            except ImportError as exc:
+                raise ImportError(
+                    "Could not import sentence_transformers or langchain package. "
+                    "Please install with `pip install sentence_transformers langchain`."
+                ) from exc
+
+            embed_model = LangchainEmbedding(HuggingFaceEmbeddings())
```

Comment:
Can we explicitly put the model name here? Just so that any default changes in langchain don't catch us off-gaurd

Reply:
sounds good. We can make it a const `DEFAULT_HUGGINGFACE_EMBEDDING_MODEL`

Reply:
Let's use `all-MiniLM-L6-v2 ` as the default due to the higher throughput (more than 5x). See: https://www.sbert.net/docs/pretrained_models.html